_staging.yml

```diff
@@ -0,0 +1,3 @@
+
+url: https://analytics-staging.app.cloud.gov
+data_url: https://analytics-staging.app.cloud.gov/data
```

Comment:
When building and deploying for staging, the Jekyll build will point the site and data URLs at its own cloud.gov URL.